helm_deploy/values-prod.yaml

```diff
@@ -4,6 +4,23 @@
 generic-service:
   ingress:
     host: approved-premises-api.hmpps.service.justice.gov.uk
+    tlsSecretName: hmpps-approved-premises-api-prod-cert
+
+  env:
+    SPRING_PROFILES_ACTIVE: prod
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
+    HMPPS_AUTH_URL: https://sign-in.hmpps.service.justice.gov.uk/auth
+    SERVICES_COMMUNITY-API_BASE-URL: https://community-api.delius.probation.hmpps.dsd.io # assumed value
```

Comment:
@edavey When you have a moment I wonder if you could do another check on these assumed urls for us please. They aren't in the cloud-platform-environments repo so I'm not sure where to check to confirm these.

Reply:
I think Community API is incorrect. See https://github.com/ministryofjustice/make-recall-decision-api/blob/60ae20013a5bd039374c83b918490831f100a1b5/helm_deploy/values-prod.yaml#L12

Reply:
Thanks Ed. That has been updated and I'm assuming `SERVICES_PRISONS-API_BASE-URL` and `SERVICES_AP-OASYS-CONTEXT-API_BASE-URL` were both correct.